qemu/tests/cfg/check_block_size.cfg

```diff
@@ -69,3 +74,8 @@
             nic_mode = tap
             physical_block_size_stg = 512
             logical_block_size_stg = 512
+
+    4096_4096..ovmf:
+        need_install = yes
+    extra_cdrom_ks..default_bios:
+        no 4096_4096
```

Comment:
I thought you could put line 78 ~ 79 into the variants `- 4096_4096` and line 80 ~ 81 into  the variants `- extra_cdrom_ks`.

Reply:
please check latest update